src/functions.js

```diff
@@ -1,30 +1,49 @@
 function removeMiddle( words ){
   // words is an array which contains an odd number of strings
+  
   // return a new array containing only the middle word
   // the words array should no longer contain the middle word
   // hint: splice
+  return words.splice(Math.ceil(words.length / 2), 1);
```

Comment:
Quite close to the solution... Anyway `Math.ceil` will return [the smallest integer greater than or equal the given number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil), (i.e. given five, it will return 3). That would be great if array would start indexing from 1, but as you know, arrays are indexed from 0. So try to look for another function from the {Math object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math)